x-pack/plugins/ml/public/application/datavisualizer/datavisualizer_selector.tsx

```diff
@@ -134,7 +134,7 @@ export const DatavisualizerSelector: FC = () => {
               title={
                 <FormattedMessage
                   id="xpack.ml.datavisualizer.selector.selectDataViewTitle"
-                  defaultMessage="Select a data view"
```

Comment:
Later in this file, the following description might also need editing:
>  defaultMessage="Visualize the data in an existing Elasticsearch index."

According to the glossary, data views can be one or more data streams, indices, or aliases. Therefore the reference to "an... Elasticsearch index" is misleading. Since this sentence doesn't really add any new information anyway, is it possible to omit it entirely?

Reply:
Done in https://github.com/elastic/kibana/pull/123019/commits/5e903277b71de4ea1b8dbea10d25dccfe1d6a83b